rapid7_insightvm/komand_rapid7_insightvm/util/resource_helper.py

```diff
@@ -71,13 +37,15 @@ class ResourceHelper(object):
     the __init__ function during instantiation.
     """
 
-    _ERRORS = {
-        400: "Bad Request",
-        401: "Unauthorized",
-        404: "Not Found",
-        500: "Internal Server Error",
-        503: "Service Unavailable",
-        000: "Unknown Status Code"
+    # Currently only handling the most common requests exceptions more can be added as needed
+    Request_exceptions = {
+        requests.HTTPError: '',
+        requests.ConnectionError: '',
+        requests.Timeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
+        requests.ConnectTimeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
+        requests.ReadTimeout: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
+        requests.TooManyRedirects: 'Ensure proper network connectivity between the orchestrator and the IVM consul',
+        'Unhandled exception': 'contact support for assistance'
```

Comment:
```suggestion
        'Unhandled exception': 'contact support for assistance'
```
Turn this `Unhandled exception` into a string constant for referencing across the rest of this file. Using "magic strings" like this is very error prone.